libs/service-portal/assets/src/screens/VehicleBulkMileage/VehicleBulkMileageFileDownloader.tsx

```
@@ -32,17 +32,23 @@ const VehicleBulkMileageFileDownloader = ({ onError }: Props) => {
-    </Button>
+    <Inline space={1}>
+      {(['csv', 'xlsx'] as const).map((type) => (
+        <Button
+          colorScheme="default"
+          icon="download"
+          iconType="outline"
+          size="default"
+          variant="utility"
+          onClick={() => downloadExampleFile(type)}
+          loading={isLoading}
+        >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)